### ISSUE TITLE
Strange Net output PSNR?

### ISSUE THREAD

**Issue by cscyangj** [jmiller656/EDSR-Tensorflow#26]
Why this happen?

I did not change net params, and using dataset of my own. PSNR is normal at first , while a big degrade then happed.
![psnr](https://user-images.githubusercontent.com/25877460/37633442-609d4ae2-2c2d-11e8-97d8-32fba6f6acea.png)

**Comment by hrahoy** [comment]
In my case, the lower learning rate solved this problem.

**Comment by purse1996** [comment]
After upsamle by sub-pix, there is an annotated code here output = slim.conv2d(x_reslut,3,[3,3])， it seems to  a 3x3 convlution after upsample. When I use the convolution, the same problem happen as you. When removing it, it gets normal.

**Comment by ZhenyF** [comment]
@purse1996 I meet the same problem as you: when I completely implement the upscale method in the paper (add a convolution after the upsampling layer), the output is lower than 20 but when I remove the final conv layer is goes fine.
I also tried the code using pytorch. It turns out that the paper auther's method on pytorch works pretty good.
Do you know why is that?

**Comment by purse1996** [comment]
In fact, I don't solve the problem and just ignore the convolution layer.
If you have any idea about it, please tell me. Thank you very much.
Additionally, Do you meet the issue that the gpu-util is always zeros when
using the code, Do you have any solutions? Thanks a lot.

ZhenyF <notifications@github.com> 于2018年8月26日周日 上午4:19写道：

**Comment by ZhenyF** [comment]
@purse1996 One of the reasons causes your low GPU util might be your data processing period in CPU is long, so your GPU need to wait for data from CPU.

**Comment by suke27** [comment]
seems psnr is poor during training, always less  than 15. I did not add output = slim.conv2d(x_reslut,3,[3,3])，  after upsample, anybody know what is the reason
